react/features/base/lib-jitsi-meet/actions.js

```diff
@@ -44,19 +45,22 @@ export function initLib() {
             throw new Error('Cannot init lib-jitsi-meet without config');
         }
 
-        // FIXME Until the logic of conference.js is rewritten into the React
-        // app we, JitsiMeetJS.init is to not be used for the React app.
-        if (typeof APP !== 'undefined') {
-            return Promise.resolve();
+        if (typeof APP === 'undefined') {
+            dispatch({ type: LIB_WILL_INIT });
```

Comment:
Does this mean LIB_WILL_INIT is only triggered for mobile? If so, why? I almost feel like if this promise-storing solution is to be used, re-using LIB_WILL_INIT and having that hold the promise might be a route to consider.

Reply:
Correct, LIB_WILL_INIT is only triggered for mobile. I'm trying to preserve the original logic.

I was thinking to do the same and I really like the idea. The problem is that when we get the promise the init has already started. So the name of the action will be confusing. Also there is a logic for log initialization that has to be executed before JitsiMeetJS.init.